src/control/lib/control/system.go

```diff
@@ -946,3 +946,65 @@ func PingRanks(ctx context.Context, rpcClient UnaryInvoker, req *RanksReq) (*Ran
 
 	return invokeRPCFanout(ctx, rpcClient, req)
 }
+
+// SystemCleanupReq contains the inputs for the system cleanup request.
+type SystemCleanupReq struct {
+	unaryRequest
+	msRequest
+	sysRequest
+	Machine string `json:"machine"`
+}
+
+type CleanupCount struct {
+	UUID  string `json:"uuid"`  // Unique identifier
+	Count uint32 `json:"count"` // Ranks of pool service replicas
+}
+
+// SystemCleanupResp contains the request response.
+type SystemCleanupResp struct {
+	Status int32
```

Comment:
shouldn't this have a json tag? I think Mike is pushing for all the exported stuct fields to be tagged.

Reply:
It should and I added one. I based this off of ListPoolsResp I believe which does not have a json tag for status.